backend/src/main/java/gov/cdc/usds/simplereport/service/errors/RestAuditFailureException.java

```diff
@@ -0,0 +1,19 @@
+package gov.cdc.usds.simplereport.service.errors;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * An error indicating that the audit logging step failed, and should not be retried. Since we do
+ * not care to disclose this information, and the most likely cause is a maliciously crafted request
+ * (unless the database went down at a very, very precise moment in time), we instruct the servlet
+ * to respond with {@link HttpStatus#BAD_REQUEST}.
+ */
+@ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
Super nitpicky and ignorable -- shouldn't this be a 500?

Reply:
I went with 400 on the grounds that the most likely reason for a failure was some manner of intentionally-crafted nutty request. That is admittedly speculative on my part, though—it definitely _is_ an internal server error, so 500 (or rather, leaving it out and letting the servlet default to that) was my initial idea. I'm not convinced either way right now.